rasa/nlu/featurizers/dense_featurizer/spacy_featurizer.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+import typing
+from typing import Any, Optional, Dict, Text
+
+from rasa.nlu.config import RasaNLUModelConfig
+from rasa.nlu.featurizers.featurzier import Featurizer
+from rasa.nlu.training_data import Message, TrainingData
+
+if typing.TYPE_CHECKING:
+    from spacy.tokens import Doc
+
+from rasa.nlu.constants import (
+    TEXT_ATTRIBUTE,
+    SPACY_DOCS,
+    DENSE_FEATURE_NAMES,
+    DENSE_FEATURIZABLE_ATTRIBUTES,
+    TOKENS_NAMES,
+    CLS_TOKEN,
+)
+
+
+class SpacyFeaturizer(Featurizer):
```

Comment:
I think component loading is done through classpaths, so moving them will def break model compatibility

Reply:
yes, the code is model breaking, also for other reasons